packages/mantine/src/templates/ButtonTemplates/IconButton.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
Rather than importing `React` can you instead import the elements of react instead?
```suggestion
import { MouseEventHandler } from 'react';
```